homeassistant/components/onewire/sensor.py

```diff
@@ -246,9 +277,13 @@ def get_entities(onewirehub: OneWireHub, config):
             device_type = device["type"]
             device_id = os.path.split(os.path.split(device["path"])[0])[1]
             dev_type = "std"
+            device_path = device["path"]
             if "EF" in family:
                 dev_type = "HobbyBoard"
                 family = device_type
+            if "7E" in family:
+                dev_type = "EDS00xx"
+                family = onewirehub.owproxy.read(f"{device_path}device_type").decode()
 
             if family not in hb_info_from_type(dev_type):
```

Comment:
Please rename function, as it is not longer hobbyboard specific (and it doesn't need a default value).

```suggestion
            if family not in get_sensor_types(device_sub_type):
```